projects/xerces-c/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER vincent.ulitzsch@live.de
+RUN apt-get update && apt-get install -y make autoconf automake libtool wget zlib1g-dev
+RUN apt-get update && \
+apt-get install -y  libtool ninja-build cmake subversion
+RUN apt-get install -y libstdc++6
```

Comment:
Why not install all packages on one line?

Reply:
Addressed in latest commit.